src/posthog-core.js

```diff
@@ -1527,6 +1527,53 @@ PostHogLib.prototype.clear_opt_in_out_captureing = function (options) {
     this.clear_opt_in_out_capturing(options)
 }
 
+/**
+ * Integrate Sentry with PostHog. This will add a direct link to the person in Sentry, and an $exception event in PostHog
+ *
+ * ### Usage
+ *
+ *     Sentry.init({
+ *          dsn: 'https://example',
+ *          integrations: [
+ *              new posthog.SentryIntegration(posthog)
+ *          ]
+ *     })
+ *
+ * @param {Object} [posthog] The posthog object
+ * @param {string} [organization] Optional: The Sentry organization, used to send a direct link from PostHog to Sentry
+ * @param {Number} [projectId] Optional: The Sentry project id, used to send a direct link from PostHog to Sentry
+ */
+PostHogLib.prototype.sentry_integration = function (_posthog, organization, projectId) {
+    this.setupOnce = function (addGlobalEventProcessor) {
+        // Poll for PostHog to load
+        function poll() {
+            if (!_posthog.__loaded) {
+                setTimeout(poll, 200)
+            } else {
+                Sentry.setTag('PostHog URL', _posthog.config.api_host + '/person/' + _posthog.get_distinct_id())
+            }
+        }
+        poll()
+        addGlobalEventProcessor((event) => {
+            if (event.level !== 'error') return event
+            let data = {
+                $sentry_event_id: event.event_id,
+                $sentry_exception: event.exception,
```

Comment:
Q: Should we include the library here? e.g. `$library: 'posthog-js'`?

Reply:
We automatically send that for each event!